Documentation/concepts/ebpf/lifeofapacket.rst

```diff
@@ -45,54 +45,6 @@ the normal flow.
 
 .. image:: _static/cilium_bpf_ingress.svg
 
-veth-based versus ipvlan-based datapath
-=======================================
-
-.. include:: ../../tech-preview.rst
-
-By default Cilium CNI operates in veth-based datapath mode which allows for
-more flexibility in that all BPF programs are managed by Cilium out of the host
-network namespace such that containers can be granted privileges for their
-namespaces like CAP_NET_ADMIN without affecting security since BPF enforcement
-points in the host are unreachable for the container. Given BPF programs are
-attached from the host's network namespace, BPF also has the ability to take
-over and efficiently manage most of the forwarding logic between local containers
-and host since there always is a networking device reachable. However, this
-also comes at a latency cost as in veth-based mode the network stack internally
-needs to be re-traversed when handing the packet from one veth device to its
-peer device in the other network namespace. This egress-to-ingress switch needs
-to be done twice when communicating between local Cilium endpoints, and once
-for packets that are arriving or sent out of the host.
```

Comment:
Wouldn't it make sense to keep this paragraph (s/By default//)?

Reply:
Would likely make sense, but probably asking for a bigger rework, also describing changes we did in https://cilium.io/blog/2020/11/10/cilium-19#veth to not have it stay just as is w/o mentioning these optimizations.